lib/dl_core_testing/dl_core_testing/connection.py

```diff
@@ -80,7 +76,13 @@ def make_saved_connection(
 ) -> ConnectionBase:
     conn = make_connection(us_manager=sync_usm, conn_type=conn_type, conn_name=conn_name, data_dict=data_dict)
     sync_usm.save(conn)
-    return conn
+    conn_id = conn.uuid
+    if conn_id is None:
+        raise ValueError("Connection ID is None")
+
+    conn_us = sync_usm.get_by_id(conn_id)
+    assert isinstance(conn_us, ConnectionBase)  # for typing
```

Comment:
for typing only you can use `typing.cast(...)`

Reply:
Adopting best practices))

https://github.com/datalens-tech/datalens-backend/blob/be2b4de77678102324f62dfbf332c3c37e23cf43/lib/dl_api_lib/dl_api_lib/app/control_api/resources/connections.py#L94

Reply:
who said it is?

Reply:
Just a joke)
Fixed!